README.md

```diff
@@ -12,3 +12,13 @@ $ <docker build | buildah bud> --file pulp_ci/Containerfile --tag pulp/pulp-ci:l
 $ <docker build | buildah bud> --file pulp_fedora31/Containerfile --tag pulp/pulp-fedora31:latest .
 $ <docker build | buildah bud> --file pulp_galaxy_ng/Containerfile --tag pulp/pulp-galaxy-ng:latest .
 ```
+
+## Specifying versions
+
+By default, containers get built using the latest version of each Pulp component. If you want to
+specify a version of a particular component, you can do so with args:
+
+```bash
+$ <docker build | buildah bud> --build_arg PULPCORE_VERSION="==3.5.0" --file pulp_fedora31/Containerfile
```

Comment:
Is this collecting compatible versions of all the plugins in that case, or do you actually need to specify them all at this point?

Reply:
BTW: Thank you for adding explicit docs around this.

Reply:
I tested it out and ran `pip install --use-feature=2020-resolver pulpcore==3.3.0 pulp_file` which pulled down `pulp-file-0.3.0` so I think pip will handle it.